LFGBulletinBoard/GroupBulletinBoard.lua

```diff
@@ -545,26 +545,14 @@ function GBB.Init()
 			_, GBB.DB.AnnounceChannel = GetChannelList()
 		end
 	end
+	
+	---@type EditBox # making this local isnt required, just here for the luals linter
+	local GroupBulletinBoardFrameResultsFilter = _G["GroupBulletinBoardFrameResultsFilter"];
+	GroupBulletinBoardFrameResultsFilter:SetFontObject(GBB.DB.FontSize);
+	GroupBulletinBoardFrameResultsFilter:SetHeight(GroupBulletinBoardFrameTitle:GetHeight());
+	GroupBulletinBoardFrameResultsFilter:SetTextColor(1, 1, 1, 1);
+	GroupBulletinBoardFrameResultsFilter:HookScript("OnTextChanged", GBB.UpdateList);
```

Comment:
My biggest concern here is not having a debounce on the call to `GBB.UpdateList` meaning it will get called for every keystroke in the editbox.